src/pages/SinglePracticePage/SinglePracticePage.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Footer from 'src/components/Footer';
+import HeroPractice from 'src/components/HeroPractice';
+import LinkBox from 'src/components/LinkBox';
+import { LinkBoxContainer } from 'src/components/LinkBox/styled';
+import ShortcutLink from 'src/components/ShortcutLink';
+import { ShortcutLinkContainer } from 'src/components/ShortcutLink/styled';
+import { Icons } from 'src/config/icons';
+import { Heading, SinglePracticeWrapper } from './styled';
+
+function SinglePracticePage() {
+  return (
+    <SinglePracticeWrapper>
+      <HeroPractice
+        className="single-practice-page__hero-practice"
+        linkText={'Link to practices'}
```

Comment:
Need to run Prettier, string props shouldn't need to escape JSX 😅 
```suggestion
        linkText="Link to practices"
```